homeassistant/components/mazda/sensor.py

```diff
@@ -41,92 +41,142 @@ class MazdaSensorEntityDescription(SensorEntityDescription):
     ] = lambda data, unit_system: None
 
 
+def _get_distance_unit(unit_system):
+    return (
+        LENGTH_MILES
+        if unit_system.name == CONF_UNIT_SYSTEM_IMPERIAL
```

Comment:
We prefer to avoid a ternary operator expression if it spans more than one line. Use a regular if else block.

Reply:
Fixed.